docs/GettingStarted.rst

```diff
@@ -1290,8 +1286,16 @@ The action is successfully done after confirming it as for the two options above
 
 You may display the home page to verify that the news is available in the appropriate block.
 
+You may display one of the following type of blocks :
```

Comment:
Not sure about the term blocks. Here we are talking about different views for the same information. So I would say something like : "You can display the pinned articles with one of the following layouts:". And update "block" by "layout" in the list. WDYT ?

Reply:
Ok. Changes done.